utils/debian_1_create.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+echo ================================
+echo === Generating debian folder ===
+echo ================================
+
+#ROOTDIR=buster
+ROOTDIR=$1
+
+cd out/linux64/build
+
+sudo debootstrap --arch amd64 --no-check-gpg $ROOTDIR $ROOTDIR http://http.debian.net/debian/
+
+sudo mkdir -p $ROOTDIR/home/root
+
+CHROOTCMD="eval LC_ALL=C LANGUAGE=C LANG=C sudo chroot $ROOTDIR"
+
+$CHROOTCMD apt clean
+$CHROOTCMD apt update
+$CHROOTCMD apt install -y wpasupplicant rfkill vim ssh sudo connman parted cloud-guest-utils
+
+sudo cp -r tmp/deploy/deb $ROOTDIR/tmp/
+
+$CHROOTCMD dpkg -i /tmp/deb/edison/kernel-image-bzimage-5.2.0-edison-acpi-standard_5.2.0-r0_amd64.deb
+$CHROOTCMD dpkg -i /tmp/deb/edison/kernel-image-5.2.0-edison-acpi-standard_5.2.0-r0_amd64.deb
+$CHROOTCMD dpkg -i /tmp/deb/edison/kernel-5.2.0-edison-acpi-standard_5.2.0-r0_amd64.deb
+$CHROOTCMD dpkg -i /tmp/deb/edison/kernel-module-btbcm-5.2.0-edison-acpi-standard_5.2.0-r0_amd64.deb
+$CHROOTCMD dpkg -i /tmp/deb/edison/kernel-module-libcomposite-5.2.0-edison-acpi-standard_5.2.0-r0_amd64.deb
+$CHROOTCMD dpkg -i /tmp/deb/edison/kernel-module-u-serial-5.2.0-edison-acpi-standard_5.2.0-r0_amd64.deb
+$CHROOTCMD dpkg -i /tmp/deb/edison/kernel-module-mmc-core-5.2.0-edison-acpi-standard_5.2.0-r0_amd64.deb
+$CHROOTCMD dpkg -i /tmp/deb/edison/kernel-module-brcmutil-5.2.0-edison-acpi-standard_5.2.0-r0_amd64.deb
+$CHROOTCMD dpkg -i /tmp/deb/edison/kernel-module-brcmfmac-5.2.0-edison-acpi-standard_5.2.0-r0_amd64.deb
+$CHROOTCMD dpkg -i /tmp/deb/edison/kernel-module-iptable-nat-5.2.0-edison-acpi-standard_5.2.0-r0_amd64.deb
+$CHROOTCMD dpkg -i /tmp/deb/edison/kernel-module-hci-uart-5.2.0-edison-acpi-standard_5.2.0-r0_amd64.deb
+$CHROOTCMD dpkg -i /tmp/deb/edison/kernel-module-rfcomm-5.2.0-edison-acpi-standard_5.2.0-r0_amd64.deb
+$CHROOTCMD dpkg -i /tmp/deb/edison/kernel-module-intel-soc-pmic-mrfld-5.2.0-edison-acpi-standard_5.2.0-r0_amd64.deb
```

Comment:
I will try to install all modules. I did not do it as I was afraid of some conflicts, but I will give it a try.

Reply:
Our initramfs kernel still loads the acpi tables. In principle it can load all provided modules too.
Just some initialization will be missing like:
- https://github.com/htot/meta-intel-edison/blob/master/meta-intel-edison-bsp/recipes-support/gadget/gadget_0.1.bb
- https://github.com/htot/meta-intel-edison/blob/master/meta-intel-edison-bsp/recipes-support/pwr-button-handler/pwr-button-handler_0.1.bb

Gadget can provide an ethernet connection via the usb port, it will probably not cause conflics as it depends on bash only.